src/components/ItaliaTheme/Blocks/Listing/SliderTemplate.jsx

```diff
@@ -81,7 +81,7 @@ function NextArrow(props) {
       onKeyDown={handleKeyboardUsers}
       id="sliderNextArrow"
     >
-      <Icon icon="chevron-right" key="chevron-right" />
+      <Icon icon="chevron-right" key="chevron-right" title="sliderNextArrow" />
```

Comment:
Non vorrei dire una cavolata ma: vedo che in generale sono misti tra tradotti con intl e title=id a caso. Non sarebbe meglio metterli tutti tradotti? Dovresti avere gia' il 98% dei messaggi, usa quelli degli span visually-hidden o del primo pezzo di testo contiguo che rappresenta la stessa cosa